### PR TITLE
initial

### PR DESCRIPTION
**Pull request by SF2311**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+    dest: /usr/local/bin/efi-clone.sh
+    mode: 0755
 
+- name: Create Service from template
+  template:
+    src: efi-clone.service.j2
+    dest: /etc/systemd/system/efi-clone.service
+  notify: reload systemd
+
+
```
Comment (neumantm) [inline tasks/main.yml:12]: Why two empty lines?

File: handlers/main.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+- name: reload systemd
+  systemd:
+    daemon_reload: yes
+    name: efi-clone.service
+    state: reloaded
```
Comment (neumantm) [inline handlers/main.yaml]: I don't think this does anything with the given service file, does it?

File: tasks/main.yml (hunk 1 above)
Comment (SF2311) [reply tasks/main.yml:12]: That's just a formatting error. I've fixed it.

File: handlers/main.yaml (hunk 2 above)
Comment (SF2311) [reply handlers/main.yaml]: Yes, you are right. I reread the systemd docs and enabling a service also does a reload, so this is not necessary.
I removed the handler.

Comment (neumantm) [reply handlers/main.yaml]: That is not what I meant.
I thought the `daemon_reload` was necessary, but the `state: reloaded` not (as that tells the service to reload its config file or such, but our service does not implement this).
Can you provide a link to the docs that explain that `daemon_reload` is not required?

Comment (SF2311) [reply handlers/main.yaml]: From `man systemctl` in the section for `enable:`

> After the symlinks have been
           created, the system manager configuration is reloaded (in a
           way equivalent to daemon-reload), in order to ensure the
           changes are taken into account immediately.

**Review by neumantm — APPROVED** [review]

**Pull request merged**